vm/systemSmartContracts/staking.go

```diff
@@ -252,12 +335,17 @@ func (r *stakingSC) unJail(args *vmcommon.ContractCallInput) vmcommon.ReturnCode
 			return vmcommon.OutOfGas
 		}
 
+		if stakedData.UnJailedNonce <= stakedData.JailedNonce {
```

Comment:
not the other way around? jailed should be smaller than unjailed to removeFromJailed?

Reply:
if you already unjailed yourself a few moments ago there is no need to remove that again.

Reply:
ok-ish... but afterwards you bleed the stake 2-3-4 times how many times it enters?